packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -307,6 +307,11 @@ export interface DeployOptions {
    * AWS SDK
    */
   sdk: ISDK;
+
+  /**
+   * Whether to execute or not the ChangeSet
```

Comment:
could probably be simplified to say `Whether to execute the ChangeSet`
Please add an `@default` and describe the behaviour when the optional parameter is not set

Reply:
Done.